nettests/facebook_messenger.go

```diff
@@ -1,15 +1,11 @@
-package im
-
-import (
-	"github.com/ooni/probe-cli/nettests"
-)
+package nettests
```

Comment:
All nettests have been flattened into the root nettests package. I think eventually we should move all the nettest specific code into probe-engine as there is currently so many levels of indirection going on here.

Reply:
Yes, I believe we should aim for slowly moving code in probe-engine.